scripts/core/ui/components/Form/Row.tsx

```diff
@@ -1,23 +1,40 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
 import './style.scss';
 
+interface IProps {
+    flex?: boolean;
+    noPadding?: boolean;
+    className?: string | {[classname: string]: boolean};
+    halfWidth?: boolean;
+    enabled?: boolean;
+    'data-test-id'?: string;
+}
+
 /**
  * @ngdoc react
  * @name Row
  * @description Component to encapsulate a component in Form row style
  */
-export const Row: React.StatelessComponent<any> = ({children, flex, noPadding, halfWidth, className, enabled}) => (
-    !enabled ? null : (
+export const Row: React.FC<IProps> = ({
+    children,
+    flex,
+    noPadding,
+    halfWidth,
+    className,
+    enabled,
+    'data-test-id': testId,
+}) => (
+    enabled === false ? null : (
```

Comment:
@thecalcc In the original implementation, `enabled` defaulted to `true`. To keep the changes small without improving this (like using a `disabled` prop instead), I am just testing is false for now until we re-write these forms.